nx-X11/programs/Xserver/Imakefile

```diff
@@ -316,7 +316,6 @@ $(NXAGENTOBJS) $(NXAGENTLIBS) $(NXAGENTSYSLIBS):: $(NXAGENTDIRS)
 #if defined(SunArchitecture)
 NXAGENTNXLIBS = -L/usr/sfw/lib \
                 -L../../../nxcomp  \
-                -L../../../nx-X11/exports/lib \
```

Comment:
`-L../../exports/lib` is being inserted by `LdPreLib`. Took me a while to figure that out.